setup.md

```diff
@@ -58,9 +58,7 @@ devtools::install_github("rossdwyer/VTrack")
 # glatos - acoustic telemetry package that does filtering, vis, array simulation, etc.
 install.packages('remotes')
 library(remotes) 
-install_url("https://gitlab.oceantrack.org/GreatLakes/glatos/-/raw/act-workshop-2021/glatos_0.4.2.1.tar.gz",
-              build_opts = c("--no-resave-data", "--no-manual"))  
-
+remotes::install_github('ocean-tracking-network/glatos')
```

Comment:
We're keeping this general, but there's a special version of glatos for this workshop, right? Which incorporates all our fixes? Where do we say to install that?

Reply:
In whichever fork for whichever workshop. in OTW we want generic.